docs/security/how-to-authenticate.md

```diff
@@ -0,0 +1,176 @@
+---
+title: "How to authenticate"
+slug: /security/how-to-authenticate
+keyword: security authentication oauth kerberos
+license: "This software is licensed under the Apache License version 2."
+---
+
+## Authentication
+
+Apache Gravitino supports three kinds of authentication mechanisms: simple,OAuth and Kerberos.
+
+### Simple mode
+
+Simple mode is the default authentication option of the server.
+
+For the client side, if it doesn't set the authentication explicitly, it will use anonymous to access the server.
```

Comment:
I'm confused about this,  if user doesn't set   authentication, it will be simple mode, the user is `anonymous` or from `GRAVITINO_USER` ?

Reply:
if users do not set the authentication explicitly, they will use the simple mode to access the Gravitino server and the corresponding user name is `anonymous`.

Reply:
the logic is too odd

Reply:
Many systems have similar logic. If user doesn't enable authentication, the user uses `anonymous`.

Reply:
yes, but you said the default authentication is `simple` in which the user is not `anonymous`

Reply:
If request don't have credential, simple mode think it's anonymous.